py3status/modules/spaceapi.py

```diff
@@ -57,10 +52,9 @@ def check(self):
                 self.closed_color = ''
 
             # grab json file
-            json_file = urllib.request.urlopen(self.url)
-            reader = codecs.getreader("utf-8")
-            data = json.load(reader(json_file))
-            json_file.close()
+            data = self.py3.get_json_data(self.url)
+            if data is None:
+                self.py3.notify_user('{} cannot be reached.'.format(self.url))
```

Comment:
I'll set the error short and full text to "Unavailable" and "Site Unavailable", respectively.  At least for now.